### PR TITLE
Added personal resume

### PR DESCRIPTION
**Pull request by AshbyCollado**
Ashby (Frasier) Resume

### CODE REVIEW

**Review by its-anurup — APPROVED** [review]
ok, i approve. thats what i had to do right ?

**Review by wlfmnstr — APPROVED** [review]
lgtm

**Comment by wlfmnstr** [comment]
> ok, i approve. thats what i had to do right ?

Close @its-anurup ! - Extra reviews are always good! In this case, my repository has permissions configured such that only once a code owner (me) has approved a PR can it be merged :)

**Pull request merged**
